### PR TITLE
feat: recommendationservice crash recipe

### PR DESCRIPTION
**Pull request by daniel-sanche**
Added a new SRE recipe that triggers a crash in the recommendation service. This will show an HTTP500 page on the frontend, and trigger obvious error messages in the logs, making it a good candidate for future Qwiklabs

I also added basic tests using bash scripts, triggered in our E2E test pipeline

### CODE REVIEW

**Comment by minherz** [comment]
@daniel-sanche can you have a look into an error of the e2e test? the error claims that it cannot find `.github/workflow/e2e_scripts/clean.sh`. It might be related to recent changes that were done to the repo.

File: sre-recipes/recipes/recommendation_crash_recipe/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Google LLC
```
Comment (simonz130) [inline sre-recipes/recipes/recommendation_crash_recipe/__init__.py:1]: perhaps, 2021?

File: sre-recipes/recipes/recommendation_crash_recipe/recommendation_crash_recipe.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
+# Copyright 2020 Google LLC
```
Comment (simonz130) [inline sre-recipes/recipes/recommendation_crash_recipe/recommendation_crash_recipe.py:1]: 2021

**Review by simonz130 — APPROVED** [review]
Excellent work!

**Pull request merged**